src/index.ts

```diff
@@ -185,14 +184,17 @@ export class XrpSettlementEngine {
     await next()
   }
 
-  async notifySettlement (accountId: string, amount: string) {
-    const url = `${this.connectorUrl}\\accounts\\${accountId}\\settlement`
+  async notifySettlement(accountId: string, amount: Number, txHash: String) {
+    const url = `${this.connectorUrl}\\accounts\\${accountId}\\settlements`
+    debug("Sending settlement for", amount)
     const message = {
-      amount,
-      scale: 6
```

Comment:
Why have you taken out the scale?
This is the scale of the SE and the Connector may need to convert this amount if it is using a different scale internally.